source/index.html

```diff
@@ -1,12 +1,89 @@
 <!DOCTYPE html>
 <html lang="ru">
+
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Кэт энерджи</title>
   </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS, уровень 2</a>».</p>
 
+  <body>
+    <header class="header">
+      <a class="header__logo" href="">
+        <img class="header__logo-image" src="https://via.placeholder.com/191x39" width="191" height="39"
+          alt="Логотип Кэт энерджи">
+      </a>
+      <nav class="nav">
```

Comment:
```suggestion
      <nav class="header__nav nav">
```
Для того чтобы ты мог управлять расположением навигации, с помощью header__nav, а не nav напрямую.